src/styles/blocks/card.scss

```diff
@@ -0,0 +1,109 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  padding: 17px;
+  padding-top: 33px;
+  padding-bottom: 18px;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  transition: transform 0.3s;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__img {
+    height: 134px;
+    width: 160px;
+
+    margin-left: auto;
```

Comment:
To center the image, it's more common to use `margin: 0 auto;` instead of specifying `margin-left` and `margin-right` separately. This makes it clear that the intention is to center the element horizontally.